pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -275,7 +275,9 @@ type PoolSpec struct {
 }
 
 type Status struct {
-	Phase string `json:"phase,omitempty"`
+	Phase     string  `json:"phase,omitempty"`
+	AccessKey *string `json:"accessKey"`
```

Comment:
Instead of adding the content of the secret in this struct, we will reveal the user creds, which is not desired. We just want to store the k8s secret name so that the admin knows where to find the secret.

Reply:
Even I thought the same, we already store this info as `secret`, So IMO it is better store the name of the `secret` in the object store CRD

Reply:
Also, you need to use a different Status struct since this one is used by other CRDs too. Create a new one like 

```
Status            *ObjectStoreUserStatus `json:"status"`
```

Reply:
> Also, you need to use a different Status struct since this one is used by other CRDs too. Create a new one like
> 
> ```
> Status            *ObjectStoreUserStatus `json:"status"`
> ```

Done.